services/src/atdd-staging/src/main/java/org/openkilda/atdd/staging/steps/FlowCrudSteps.java

```diff
@@ -138,6 +142,24 @@ public void flowsWithAlternatePaths(int flowsAmount, int alternatePaths, int bw)
         flowIsls.putAll(flowManager.createFlowsWithASwitch(flowsAmount, alternatePaths, bw));
         //temporary resaving flows before refactoring all methods to work with topologyUnderTest
         flows = flowIsls.keySet();
+        flows.forEach(flow -> flowPaths.add(northboundService.getFlowPath(flow.getId())));
```

Comment:
You cannot get flows' path because flows are not created at this point yet. This should break the FlowCrud scenario

Reply:
It looks like a flow is already created at this point. We are creating the flow at L142.

Reply:
oh right, I thought it was added in the `flows defined over active switches in the reference topology` step